llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9764,23 +9784,36 @@ preparePlanForEpilogueVectorLoop(VPlan &Plan, Loop *L,
     Value *ResumeV = nullptr;
     // TODO: Move setting of resume values to prepareToExecute.
     if (auto *ReductionPhi = dyn_cast<VPReductionPHIRecipe>(&R)) {
+      auto *RdxResult =
+          cast<VPInstruction>(*find_if(ReductionPhi->users(), [](VPUser *U) {
+            auto *VPI = dyn_cast<VPInstruction>(U);
+            return VPI &&
+                   (VPI->getOpcode() == VPInstruction::ComputeReductionResult ||
+                    VPI->getOpcode() == VPInstruction::ComputeFindLastIVResult);
```

Comment:
Does this need to also include VPInstruction::ComputeAnyOfResult? The tests seem to be passing so I'm a bit confused. Or do AnyOf reductions still somehow end up emitting a ComputeReductionResult in some cases?

Reply:
Yep added, it looks like this by accident returned the correct result for the tests w/o sanitizers enabled.